src/Symfony/Component/Debug/ExceptionHandler.php

```diff
@@ -411,4 +414,27 @@ private function getSymfonyGhostAsSvg()
     }
+
+    private static function getTraceClassname($trace)
+    {
+        if (empty($trace['file'])) {
+            return null;
+        }
+
+        $vendorRoot = static::getVendorRoot();
+        $traceFileRelative = str_replace($vendorRoot, '', realpath($trace['file']));
+
+        if (strpos($traceFileRelative, '/vendor/') === 0) {
```

Comment:
the generic logic to get vendors and composer root is a bit more involving, see `LinkStub::getComposerRoot()`:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/VarDumper/Caster/LinkStub.php#L64